system/greenhouse-ccloud/templates/kube-monitoring-pluginpreset.yaml

```diff
@@ -15,21 +15,24 @@ metadata:
   namespace: {{ .Release.Namespace }}
 spec:
   clusterOptionOverrides:
-  {{- range $cluster, $additionalSettings := .Values.kubeMonitoring.cephClusters }}
+  {{- range $serviceName, $serviceValues := .Values.kubeMonitoring.services}}
+  {{- range $cluster, $additionalSettings := $serviceValues.clusters }}
   - clusterName: {{ $cluster }}
     overrides:
     - name: global.commonLabels
       value:
-        service: ceph
-        support_group: storage
+        service: {{ $serviceName }}
+  {{- if index $serviceValues "support_group" }}
+        support_group: {{ $serviceValues "support_group" }}
+  {{- end }}
```

Comment:
We want to enforce `service`, `support_group`, don't we? Apparently there is no `support_group` for kvm, but this needs to be added. So, I would not check if `support_group` is available, we map it and let it fail if it is not set.